stdlib/2/_io.pyi

```diff
@@ -19,91 +19,149 @@ class _IOBase(BinaryIO):
     def fileno(self) -> int: ...
     def flush(self) -> None: ...
     def isatty(self) -> bool: ...
-    def read(self, n: int = ...) -> bytes: ...
     def readable(self) -> bool: ...
-    def readline(self, limit: int = ...) -> bytes: ...
-    def readlines(self, hint: int = ...) -> list[bytes]: ...
     def seek(self, offset: int, whence: int = ...) -> int: ...
     def seekable(self) -> bool: ...
     def tell(self) -> int: ...
     def truncate(self, size: Optional[int] = ...) -> int: ...
     def writable(self) -> bool: ...
-    def write(self, s: bytes) -> int: ...
+    def __exit__(self, type, value, traceback) -> bool: ...
```

Comment:
My bad. The types got under the wheel when I shifted `__exit__` back and fro.

That said, I'm not sure what we get out of having types on `__exit__`. This function is invoked by the type-checker, not user code (nobody ever does a manual `x.__exit__(a, b,c)`), and the parameters are not originating from user code, either.

Either way, I added types.